### ISSUE TITLE
Bug in updating

### ISSUE THREAD

**Issue by cdedwards** [Donkie/Spoolman#215]
I decided to update spoolman and used your instructions for docker pull todo so. I had a fully created database of filament which was completely lost after the update. Completely back to square one. 
** UPDATE **
Found my database from the docker. Maybe a import database might be a good idea. The database I have is not recognized by spoolman (Defaults used in setup both times).

**Comment by Donkie** [comment]
can you show me your docker-compos e.yml? If you follow the README example closely this shouldn't happen.

**Comment by cdedwards** [comment]
edwards@voron:~/spools.crap $ cat docker-compose.yml 
version: '3.8'
services:
  spoolman:
    image: ghcr.io/donkie/spoolman:latest
    restart: unless-stopped
    volumes:
      - ./data:/home/edwards/.local/share/spoolman
    ports:
      - "7912:8000"
      - 
      edwards@voron:~/spools.crap $ 

I used the default setup. I found my database. But I'm unable to use it with the self-install without docker option. after performing a "strings -a" on that database I discovered it's some other data file. print data from the looks of it. The database itself of all the filaments has disappeared.
